indico/modules/events/contributions/util.py

```diff
@@ -285,7 +285,10 @@ def serialize_contribution_for_ical(contrib):
 def import_contributions_from_csv(event, f):
     """Import timetable contributions from a CSV file into an event."""
     with csv_text_io_wrapper(f) as ftxt:
```

Comment:
I think you might be able to catch the exception in there and thus cover all the places where we use this in one go

Reply:
Yep, much better than me plastering that `except` everywhere 😅